po/common/common.pot

```diff
@@ -4743,56 +4735,3 @@ msgstr "Additives"
 msgctxt "attribute_additives_setting_name"
 msgid "No or few additives"
 msgstr "No or few additives"
```

Comment:
```suggestion
msgstr "No or few additives"

msgctxt "preference_not_important"
msgid "Not important"
msgstr "Not important"

msgctxt "preference_important"
msgid "Important"
msgstr "Important"

msgctxt "preference_very_important"
msgid "Very important"
msgstr "Very important"

msgctxt "preference_mandatory"
msgid "Mandatory"
msgstr "Mandatory"

msgctxt "attribute_group_environment_name"
msgid "Environment"
msgstr "Environment"

msgctxt "attribute_ecoscore_name"
msgid "Eco-Score"
msgstr "Eco-Score"

msgctxt "attribute_ecoscore_setting_name"
msgid "Low environmental impact"
msgstr "Low environmental impact"

# keep %s, it will be replaced by the letter A, B, C, D or E
msgctxt "attribute_ecoscore_grade_title"
msgid "Eco-Score %s"
msgstr "Eco-Score %s"

msgctxt "attribute_ecoscore_a_description_short"
msgid "Very low environmental impact"
msgstr "Very low environmental impact"

msgctxt "attribute_ecoscore_b_description_short"
msgid "Low environmental impact"
msgstr "Low environmental impact"

msgctxt "attribute_ecoscore_c_description_short"
msgid "Moderate environmental impact"
msgstr "Moderate environmental impact"

msgctxt "attribute_ecoscore_d_description_short"
msgid "High environmental impact"
msgstr "High environmental impact"

msgctxt "attribute_ecoscore_e_description_short"
msgid "Very high environmental impact"
msgstr "Very high environmental impact"
```